autoload/ale/engine.vim

```diff
@@ -144,8 +144,9 @@ function! s:HandleExit(job) abort
     let g:ale_buffer_info[l:buffer].loclist = g:ale_buffer_info[l:buffer].new_loclist
     let g:ale_buffer_info[l:buffer].new_loclist = []
 
-    if g:ale_set_loclist
-        call setloclist(0, g:ale_buffer_info[l:buffer].loclist)
+    if g:ale_set_quickfix || g:ale_set_loclist
+"         call setloclist(0, g:ale_buffer_info[l:buffer].loclist)
```

Comment:
Don't comment out old code, replace it. Changes can be easily reverted with `git`.